yasmeen-alharbi/TruffleTalk
# Task/7 login view

This PR fully implements the login view and error handling. Errors show up on the relevant fields, and logging in successfully redirects the user to the home page.

**Note**: run `npm install` before testing since `axios` was installed. For testing purposes, if you login successfully, the `back to login` button on the home page will log you out. No need to logout through postman :)

# Testing
From the home page, click the login button, and you will be redirected to the login page. 
Once there:
- Try logging in without entering any details.
- Try logging in with a correct email but incorrect password.
- Try logging in with an incorrect email and any password.
- Try logging in with a correct email and password. You will be redirected to the home page.

Finally, make sure that layout looks good on web and android, as there were layout issues on web that I wasn't aware of until later.
